src/components/sessions/add-classes/DaysPicker.tsx

```diff
@@ -0,0 +1,94 @@
+import React from "react";
+
+import { makeStyles } from "@material-ui/core";
+import { ToggleButton, ToggleButtonGroup } from "@material-ui/lab";
+
+import FormRow from "components/common/form-row";
+import FieldVariant from "constants/FieldVariant";
+import QuestionType from "constants/QuestionType";
+import { DaysOfWeek } from "types";
+
+const DAYS_IN_WEEK = [
+  {
+    label: "Mon",
+    value: DaysOfWeek.Monday,
+  },
+  {
+    label: "Tue",
+    value: DaysOfWeek.Tuesday,
+  },
+  {
+    label: "Wed",
+    value: DaysOfWeek.Wednesday,
+  },
+  {
+    label: "Thu",
+    value: DaysOfWeek.Thursday,
+  },
+  {
+    label: "Fri",
+    value: DaysOfWeek.Friday,
+  },
+  {
+    label: "Sat",
+    value: DaysOfWeek.Saturday,
+  },
+  {
+    label: "Sun",
+    value: DaysOfWeek.Sunday,
+  },
+];
+
+const useStyles = makeStyles(() => ({
+  toggleButton: {
+    "&.Mui-selected": {
+      color: "#5A19B9",
+      backgroundColor: "#CCA8FF",
```

Comment:
can we add a todo comment to update the colours later? i know we have MCBP 🌈 but we gotta update override the theme as a whole later to this purple, so we can do it all at once 😄

Reply:
`:(`